nilearn/glm/tests/test_first_level.py

```diff
@@ -1798,3 +1798,29 @@ def test_missing_trial_type_column_warning(tmp_path_factory):
             "No column named 'trial_type' found" in r.message.args[0]
             for r in record
         )
+
+
+def test_first_level_from_bids_with_one_condition_missing(tmp_path_factory):
+    """One condition is missing in one events.tsv file."""
```

Comment:
This is supposed (not) to trigger something ?

Reply:
sorry I should have mention that this is not ready for review yet: opened a draft PR instead of just stashing